qfieldsync/gui/cloud_transfer_dialog.py

```diff
@@ -570,7 +656,12 @@ def _start_synchronization(self):
             assert self.project_transfer
 
             self.project_transfer.sync(
-                files["to_upload"], files["to_download"], files["to_delete"]
+                files["to_upload"],
+                files["to_download"],
+                files["to_delete"],
+                self.localized_datasets_files
+                if self.uploadLocalizedDatasetsCheck.isChecked()
+                else [],
```

Comment:
May we move this to a separate variable instead of abusing the inline if's in the argument list :)

Reply:
Note to self: Ivan doesn't like inline if's :wink: